### PR TITLE
Batch editor follow up issues

### PR DESCRIPTION
**Pull request by bp323**
**Styling**
* [x] Selected and unselected weeks have different paddings
* [x] Lecturer component active state  wreaks havoc in other batch edit input fields
timetable_administration

 * [x] When the batch edit is inactive I can click into lecturer input and get a mis-aligned autocomplete (as per functionality item below, in this state I should't be able to click inside)
timetable_administration

**Functionality**
 * [x] Add another day should add another day line item to the editor
* [x] When selecting an empty term, the batch edit is not available. It should be. 
* [x] When batch edit is inactive, the input fields and dropdowns should be disabled too

Related, on the event list:
 * [x] When am AND pm hours are displayed the date column is too short, so threedots the date/time. We should add a midge more space to the date/time column (take away from the title) to avoid this in all date/time scenarios

### CODE REVIEW

**Comment by Coenego** [comment]
I assume the colour of the 'X' might need to change to be in line with the rest of the admin colour scheme?

screen shot 2015-03-12 at 11 39 29

File: shared/gh/js/views/gh.admin-batch-edit-date.js [hunk 1]
```diff
@@ -92,14 +92,81 @@ define(['lodash', 'moment', 'gh.core', 'gh.api.config'], function(_, moment, gh,
...
+            });
+        });
+    };
+
+    /**
+     * Add another event to the terms based on the selection of weeks. If the event has already been added it won't be
      * added again unless `forceAdd` has been set to `true`
      *
      * @param {Boolean}    [forceAdd]    Whether to force adding another day. Defaults to `false`
      *
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit-date.js]: This line can possibly go

**Comment by Coenego** [comment]
Left some minor comments. Reassigning to @bp323 for follow-up

**Comment by bp323** [comment]
Followed up on review, back to @Coenego for review

**Pull request merged**